examples/bring_your_own_fips202/Makefile

```diff
@@ -49,19 +56,34 @@ ALL_SOURCE=$(MLKEM_NATIVE_SOURCE) $(FIPS202_SOURCE) $(RNG_SOURCE) $(APP_SOURCE)
 BUILD_DIR=build
 BIN=test_binary
 
+CFLAGS := \
+	-Wall \
+	-Wextra \
+	-Wmissing-prototypes \
+	-Wshadow \
+	-Wpointer-arith \
+	-Wno-long-long \
+	-Wno-unknown-pragmas \
+	-Wno-unused-command-line-argument \
+	-fomit-frame-pointer \
+	-std=c99 \
+	-pedantic \
+	-MMD \
+	$(CFLAGS)
+
 BINARY_NAME_FULL=$(BUILD_DIR)/$(BIN)
 
 $(BINARY_NAME_FULL): $(ALL_SOURCE)
 	echo "$@"
 	mkdir -p $(BUILD_DIR)
-	$(CC) $(INC) $^ -o $@
+	$(CC) $(CFLAGS) $(INC) $^ -o $@
 
 all: run
```

Comment:
Not really really related to your change, but the `all` target should be the first target in the file by convention, I think.
Can you either move that up and depend on `$(BINARY_NAME_FULL)$ or remove it entirely please. 
Same for all other examples.

Reply:
Actually, I want the default target to the 'build' target, not the 'run' target. That's what the toplevel `Makefile` also does.

I can move `all` to the top and map it to `$(BINARY_NAME_FULL)`